src/ucp/tag/rndv.c

```diff
@@ -288,16 +354,33 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_get_zcopy, (self),
                    offset, (uintptr_t)rndv_req->send.buffer % align,
                    (void*)rndv_req->send.buffer + offset, length);
 
+    if ((rndv_req->send.rndv_get.use_mrail) &&
+       ((ucp_request_is_empty_rail(&rndv_req->send.state, rndv_req->send.rndv_get.rail_idx)) ||
```

Comment:
indentation

Reply:
will fix, but current intent is more "beautiful"